src/dev/flang/fe/SourceModule.java

```diff
@@ -945,7 +946,9 @@ else if (c._hasPost != null && c._hasPostThen == null)
 A feature that does not redefine an inherited feature must not use the `redef` modifier.
     // end::fuzion_rule_PARS_NO_REDEF[]
             */
-            AstErrors.redefineModifierDoesNotRedefine(f);
+            List<FeatureAndOuter> hiddenFeaturesSameSignature = lookup(outer, f.featureName().baseName(), null, true, true);
+            hiddenFeaturesSameSignature = hiddenFeaturesSameSignature.stream().filter(fan->fan._feature.featureName().equals(f.featureName())).collect(List.collector());
```

Comment:
```suggestion
            List<FeatureAndOuter> hiddenFeaturesSameSignature = lookup(outer, f.featureName().baseName(), null, true, true)
                                   .stream().filter(fan->fan._feature.featureName().equals(f.featureName())).collect(List.collector());
```

Reply:
Instead of `fan` I think in other place `fo` (singular) is used or `fos` (plural).